test/erc_to_native/foreign_bridge.test.js

```diff
@@ -1312,4 +1590,290 @@ contract('ForeignBridge_ERC20_to_Native', async accounts => {
       })
     })
   })
+
+  describe('chai token', async () => {
+    let token
+    let foreignBridge
+    let chaiToken
+    let interestRecipient
+
+    beforeEach(async () => {
+      token = await DaiMock.new({ from: owner })
+      foreignBridge = await ForeignBridgeErcToNativeMock.new()
+      await foreignBridge.initialize(
+        validatorContract.address,
+        token.address,
+        requireBlockConfirmations,
+        gasPrice,
+        [dailyLimit, maxPerTx, minPerTx],
+        [homeDailyLimit, homeMaxPerTx],
+        owner,
+        decimalShiftZero,
+        otherSideBridge.address
+      )
+      chaiToken = await createChaiToken(token, foreignBridge, owner)
+      interestRecipient = await ERC677ReceiverMock.new()
+    })
+
+    describe('initializeChaiToken', () => {
```

Comment:
I think we should add a unit test that calls `initializeChaiToken()` two times, in order to test that it can't be initialized twice

Reply:
Sure, I will add this test